arcade/standard/bed_rush/map.xml

```diff
@@ -0,0 +1,587 @@
+        <material>air</material>
+    </offset>
+    <offset id="blue-bed-broke-2" vector="0,10,42">
+        <material>air</material>
+    </offset>
+    <all id="should-tp">
+        <variable var="round_state">20</variable> <!-- reset1-state -->
+        <participating/>
+        <match-running/>
+        <alive/>
+    </all>
+    <deny id="original-block-protection-filter">
+        <blocks region="original-block-protection">
+            <not>
+                <any>
+                    <!-- List of blocks in the original map that players can modify -->
+                    <material>air</material>
+                    <material>bed block</material>
+                    <material>sandstone</material>
+                    <material>stained clay</material>
+                </any>
+            </not>
+        </blocks>
+    </deny>
+    <deny id="red-bed-protect-1">
+        <all>
+            <blocks region="bed-red-1">
+                <all>
+                    <material>bed block</material>
+                </all>
+            </blocks>
+            <team>team-one</team>
+        </all>
+    </deny>
+    <deny id="red-bed-protect-2">
+        <all>
+            <blocks region="bed-red-2">
+                <all>
+                    <material>bed block</material>
+                </all>
+        </blocks>
+        <team>team-one</team>
+        </all>
+    </deny>
+    <deny id="blue-bed-protect-1">
+        <all>
+            <blocks region="bed-blue-1">
+                <all>
+                    <material>bed block</material>
+                </all>
+            </blocks>
+            <team>team-two</team>
+        </all>
+    </deny>
+    <deny id="blue-bed-protect-2">
+        <all>
+            <blocks region="bed-blue-2">
+                <all>
+                    <material>bed block</material>
+                </all>
+            </blocks>
+            <team>team-two</team>
+        </all>
+    </deny>
+    <all id="red-team-win">
+        <variable var="red_team_win">1</variable>
+        <team>team-one</team>
+    </all>
+    <all id="blue-team-win">
+        <variable var="blue_team_win">1</variable>
+        <team>team-two</team>
+    </all>
+    <all id="red-team-win-final">
+        <variable var="red_team_win">2</variable>
+        <team>team-one</team>
+    </all>
+    <all id="blue-team-win-final">
+        <variable var="blue_team_win">2</variable>
+        <team>team-two</team>
+    </all>
+    <all id="red-team-lose">
+        <variable var="red_team_lose">1</variable>
+        <team>team-one</team>
+    </all>
+    <all id="blue-team-lose">
+        <variable var="blue_team_lose">1</variable>
+        <team>team-two</team>
+    </all>
+    <all id="red-final">
+        <variable var="red_team_win">5</variable>
+        <team>team-one</team>
+    </all>
+    <all id="blue-final">
+        <variable var="blue_team_win">5</variable>
+        <team>team-two</team>
+    </all>
```

Comment:
ngl i don't really understand the use of red_team_win, blue_team_win, red_team_lose and blue_team_lose variables.
Why are all of those not just one team-scoped score variable (with team-specific with-team variable views for red/blue)

Reply:
Thanks, simplified down to 2 variables, team_win and team_lose with with-team views.  If I got rid of one of these variables, I'd still need to add meta logic to handle the variable reset, so it's easier keeping them distinct.